api/managementpb/backup/backups.proto

```diff
@@ -41,6 +42,20 @@ message StartBackupResponse {
   string artifact_id = 1;
 }
 
+message ListArtifactCompatibleServicesRequest {
+  // Artifact id used to determine restore compatibility.
+  string artifact_id = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+}
+
+message ListArtifactCompatibleServicesResponse {
+  repeated inventory.MySQLService mysql = 1;
```

Comment:
don't we need only one type of service? so it'll be better oneof, than everything in one message?

Reply:
sounds reasonable, will try

Reply:
so I cannot put repeated in the oneof 😞